tequilapi/endpoints/sse_handler.go

```diff
@@ -207,12 +207,13 @@ func (h *Handler) ConsumeNodeEvent(e nodeEvent.Payload) {
 }
 
 type stateRes struct {
-	NATStatus     contract.NATStatusDTO     `json:"nat_status"`
-	Services      []contract.ServiceInfoDTO `json:"service_info"`
-	Sessions      []contract.SessionDTO     `json:"sessions"`
-	SessionsStats contract.SessionStatsDTO  `json:"sessions_stats"`
-	Consumer      consumerStateRes          `json:"consumer"`
-	Identities    []contract.IdentityDTO    `json:"identities"`
+	NATStatus     contract.NATStatusDTO        `json:"nat_status"`
+	Services      []contract.ServiceInfoDTO    `json:"service_info"`
+	Sessions      []contract.SessionDTO        `json:"sessions"`
+	SessionsStats contract.SessionStatsDTO     `json:"sessions_stats"`
+	Consumer      consumerStateRes             `json:"consumer"`
+	Identities    []contract.IdentityDTO       `json:"identities"`
+	Channels      []contract.PaymentChannelDTO `json:"channels"`
```

Comment:
I think that `channels`-> `paymentChannels` would make it more self documenting

Reply:
Discussed in slack, this attribute will include broader types of channels so renaming it specifically to `payments` channel will not make much sense.